src/bundle/ui-dev/src/modules/content-tree/components/content-tree/content.tree.js

```diff
@@ -26,10 +28,46 @@ export default class ContentTree extends Component {
         this.clearDocumentResizingListeners();
     }
 
+    componentDidMount() {
+        this.containerSrollRef.addEventListener('scroll', (event) => {
+            clearTimeout(this.scrollTimeout);
```

Comment:
```suggestion
            window.clearTimeout(this.scrollTimeout);
```
To show that this is from window object